container/thread-safe/old_generic_map.go

```diff
@@ -0,0 +1,59 @@
+// Package threadsafe contains generic containers that are
```

Comment:
Thanks for the contrbution @elee1766 ! No need to keep this file around if you're just adding new methods to the map

Reply:
ok! i kept it in for the benchmark and comparison since the actual implementation of all the map methods changed.

will delete now